firmware/rhizome.ino

```diff
@@ -98,20 +111,32 @@ void setup() {
  * The meat of the program. Runs repeatedly until the Rhizome is powered off.
  */
 void loop() {
+    //probe ID finder command (switch this for the other below commands to enable)
+//    ((Ohmbrewer::Onewire*)(((Ohmbrewer::RIMS*)sprouts.front())->getTube()->getSensor()->getProbe()))
+//            ->displayProbeIds(&screen);
+//    screen.println( ((Ohmbrewer::HeatingElement*)sprouts.front())->getControlPin() );
+//    screen.println( ((Ohmbrewer::HeatingElement*)sprouts.front())->getPowerPin() );
+// int a = 0;
+//    if (a==0){
+//        sprouts.push_back(new Ohmbrewer::Thermostat(1, thermPins));
+//        a++;
+//    }
+    screen.println(thermPins->front());
+
     if((millis() - lastUpdate) > 10000) {
         // Toggle the last relay every 10 seconds, for illustration.
 //        sprouts.back()->setState(sprouts.back()->isOff()); // The last HeatingElement
 //        ((Ohmbrewer::Thermostat*)sprouts.at(0))->getElement()
 //                                               ->setState(!((Ohmbrewer::Thermostat*)sprouts.at(0))->getElement()
 //                                                                                                  ->getState()); // The Thermostat's HeatingElement
-        ((Ohmbrewer::RIMS*)sprouts.front())->getTube()
-                                           ->getElement()
-                                           ->toggleState();
+//        ((Ohmbrewer::RIMS*)sprouts.front())->getTube()
+//                                           ->getElement()
+//                                           ->toggleState();
 //        navi->publish(); // (*)
         lastUpdate = millis();
     }
-    ((Ohmbrewer::RIMS*)sprouts.front())->getTube()->getSensor()->doWork(); //Temp patch to make sensor read in TUBE
-    screen.refreshDisplay();
+//    ((Ohmbrewer::RIMS*)sprouts.front())->getTube()->getSensor()->work(); //Temp patch to make sensor read in TUBE
+//    screen.refreshDisplay();
```

Comment:
I brought this line back so I could see what's going on (assuming things don't hard fault, of course).